Gemfile

```diff
@@ -7,7 +7,7 @@ group :development do
 end
 
 group :test do
-  gem "json", '~> 1.7.7'
+  gem "json", '~> 1.8.3'
```

Comment:
I changed this because I couldn't get `1.7.7` to build with ruby 2.2+. Here is the issue that mentions the reason:

https://github.com/flori/json/issues/229